devel/create-test-data.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
 import textwrap
 
-import python_freeipa
 from faker import Faker
+import python_freeipa
```

Comment:
Are you sure? In [PEP8](https://www.python.org/dev/peps/pep-0008/#id23) they sort imports with `import x` first and `from x import y` second.

In the [Google style guide](https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#313-imports-formatting) you can see both sortings, and in the [example file](https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_google.py) that flake8-import-order provides the `import x` come before the `from x import y`.

Reply:
I've changed `import-order-style` to `smarkets` in tox.ini which doesn't require 3rd party packages to be separate from each other, but places `from ... import ...` beneath plain `import ...` statements within their respective blocks.